blog/release.md

```diff
@@ -0,0 +1,558 @@
+
+For more information, please, visit these tutorials:
+
+- [Image segmentation with `threshold()` and `fromMask()`](../docs/Tutorials/Image%20segmentation%20with%20threshold)
+- [Image segmentation with `watershed()`](../docs/Tutorials/Image%20segmentation%20with%20watershed)
+
+### Sobel and Scharr filters
+
+[Sobel](https://en.wikipedia.org/wiki/Sobel_operator), [Scharr](https://en.wikipedia.org/wiki/Sobel_operator#Alternative_operators) filters are now combined into a single `derivative()` method.
+
+```ts
+// Before
+const sobelX = img.sobelX();
+const sobelY = img.sobelY();
+
+// After
+const sobelX = img.derivative({ filter: 'sobel' });
+const sobelY = img.derivative({ filter: 'scharr' });
+```
+
+This filter now also accepts only grayscale images, since filters, like Sobel or Scharr, are used mainly on grayscale images to detect edges.
+
+### Method Renaming
+
+Several methods have been renamed for consistency:
+
+**Drawing methods**:
+
+`img.paintPolyline()` ➡️ `img.drawPolyline()`
+
+`img.paintPolygon()` ➡️ `img.drawPolygon()`
+
+`img.paintCircle()` ➡️ `img.drawCircle()`
+
+**Stack methods**
+
+`stack.getMinImage()` ➡️ `stack.minImage()`
+
+`stack.getMinImage()` ➡️ `stack.maxImage()`
+
+`stack.getAverageImage()` ➡️ `stack.meanImage()`
+
+**Pixel and value getters/setters**:
+
+`img.getBitXY()` ➡️ `mask.getBit()`
+
+`img.getBit()` ➡️ `mask.getBitByIndex()`
+
+`img.getPixelXY()` ➡️ `img.getPixel()`
+
+`img.getPixel()` ➡️ `img.getPixelByIndex()`
+
+`img.getValueXY()` ➡️ `img.getValue()`
+
+`img.getValue()` ➡️ `img.getValueByIndex()`
+
+`img.setBitXY()` ➡️ `mask.setBit()`
+
+`img.setBit()` ➡️ `mask.setBitByIndex()`
+
+`img.setPixelXY()` ➡️ `img.setPixel()`
+
+`img.setPixel()` ➡️ `img.setPixelByIndex()`
+
+`img.setValueXY()` ➡️ `img.setValue()`
+
+`img.setValue()` ➡️ `img.setValueByIndex()`
+
+**Other methods**:
+
+`img.getLocalMaxima()` ➡️ `img.getExtrema()`
+
+`img.getChannel()` ➡️ `img.extractChannel()`
+
+`img.rotateLeft()` & `img.rotateRight()` ➡️ `img.rotate()`
+
+`img.flipX()` & `img.flipY()` ➡️ `img.flip()`
+
+`img.colorDepth()` ➡️ `img.convertBitDepth()`
+
+`img.mask()` ➡️ `img.threshold()`
+
+`img.cannyEdge()` ➡️ `img.cannyEdgeDetector()`
+
+`img.blurFilter()` ➡️ `img.blur()`
+
+`img.insert()` ➡️ `img.copyTo()`
+
+`img.convolution()` ➡️ `img.directConvolution()`
+
+### Compatibility requirements
+
+- Node.js: 18+ (previously 14+)
+- TypeScript: 5.2.2+ (if using TypeScript)
+
+### Removed Features
+
+The following deprecated features have been removed:
+
+#### Images
+
+- `countAlphaPixel()` - Use custom pixel counting with `getPixel()`
+- `paintLabels()` and `roi.paint()` - Features have been removed due to dependency issues. We plan to add it back in the future updates.
+- `warpingFourPoints()` - Use `getPerspectiveWarp()` + `transform()`.
+- 32-bit color depth has been currently removed. We plan to add it back in the future updates as well.
```

Comment:
```suggestion
- 32-bit color depth has been currently removed. We plan to add it back in future updates as well.
```

Reply:
Not planning on adding 32-bit depth support in the future.